README.md

```diff
@@ -1,2 +1,30 @@
 # activemq-integration
+
 Query activemq for message age metrics and publish them
+
+# What is this plugin for?
+This pluging is for reporting messages age in the ActiveMQ-queues. It reports two metrics:
+- message.age.average - average message age in each queue
+- message.age.maximum - maximum message age in each queue
+
+# Requirements
+- Linux/Unix
+- ActiveMQ
+- Maven
+
+# Installation
+
+- clone repo
+- Edit <repo>/amq-message-age/properties file to config plugin
+- cd <repo>/amq-message-age
+- ./run.sh
+
+# "properties" file
+
+- path - path to ActiveMQ executable
+- token - SignalFX-API-TOKEN
+- sfx_host - let it be https://ingest.signalfx.com - host to send data to
+- interval - interval how often to query on ActiveMQ messages age
+- host - activemq host
+- queue_name_prifix - prefix which will be added to the queue name in the dimension defined by queue_dimension_name
```

Comment:
we have the limitation in generic jmx that we can't separate the broker from the queue or topic, but should do that here.  Have a separate dimensions for broker, queue and topic.